iceoryx_posh/test/moduletests/test_popo_untyped_subscriber.cpp

```diff
@@ -24,10 +24,15 @@
 using namespace ::testing;
 using ::testing::_;
 
+// anonymous namespace to prevent linker issues or sanitizer false positives
+// if a struct with the same name is used in other tests
+namespace
+{
 struct DummyData
 {
     uint64_t val = 42;
 };
+} // namespace
```

Comment:
I would extend this to the test fixture or even the tests themselves. Test fixture names can also cause conflicts.

Reply:
addressed in #563